<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotificationWidget</class>
 <widget class="QWidget" name="NotificationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>通知するチャンネル</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="notifyFavoriteCheckBox">
          <property name="text">
           <string>お気に入り</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="notifyNewCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>新規</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="notifyChangedCheckBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>詳細変更あり</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="favoriteGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>お気に入り通知</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="notifyChangedFavoriteCheckBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>詳細変更があったチャンネルも通知する</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>通知するチャンネルの最低スコア</string>
          </property>
          <property name="indent">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="minimumScoreSpinBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="balloonGroupBox">
     <property name="title">
      <string>バルーン通知</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="showBalloonMessageCheckBox">
        <property name="text">
         <string>システムトレイアイコンにメッセージを表示する</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>一度に表示する最大チャンネル数</string>
          </property>
          <property name="indent">
           <number>20</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="maxShowChannelsSpinBox">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="soundGroupBox">
     <property name="title">
      <string>サウンド通知</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <layout class="QHBoxLayout">
        <item>
         <widget class="QCheckBox" name="playSoundCheckBox">
          <property name="text">
           <string>ファイルを再生</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="soundFileEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectFileButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>選択...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playSoundButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>♪</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>再生可能ファイルは WAV と OGG です。(Windows は WAV のみ)</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="indent">
         <number>20</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>117</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>playSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectFileButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>soundFileEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSoundCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>playSoundButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>soundGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectFileButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>notifyFavoriteCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>favoriteGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>balloonGroupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showBalloonMessageCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxShowChannelsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showBalloonMessageCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
